taxi/projects.py

```diff
@@ -118,9 +118,13 @@ def tuple_to_str(cls, t):
 
 
 class Activity:
-    def __init__(self, id, name):
+    def __init__(self, id: str, name: str, active: bool = True):
         self.id = id
         self.name = name
+        self.active = active
+
+    def is_active(self) -> bool:
```

Comment:
While I understand this was done to mirror the `Project` API, I’m not a big fan of this method because having both `Activity.active` and `Activity.is_active` goes against the zen of Python "there should be one-- and preferably only one --obvious way to do it". So I would either:

* Rename `Activity.active` to `Activity._active` to show that it’s a private member
* Remove `Activity.is_active()`, rename `Activity.active` to `Activity.is_active`, and make `Project.is_active()` a property

I don’t have a preference so… your call. ;)

Reply:
Nice catch, thanks.